### ISSUE TITLE
Support more paragraph styles

### ISSUE THREAD

**Issue by AntonyBoucher** [TorstenDittmann/OmniaWrite#68]
OW currently supports ordinary paragraphs (\<p>), quotes (\<q>) and code blocks (\<code>). Some other styles may also be useful in practice:

* **Bullet list** (\<ul>)

* **Ordered list** (\<ol>)

* **Left/centered/right justified paragraph**

* **Indented paragraph**: a paragraph with extra margin, for example a letter or note being transcribed within a novel. From Lovecraft's *The Whisperer in Darkness*:

    ![image](https://user-images.githubusercontent.com/69065091/90339863-2f0fee00-dfb1-11ea-987b-34a017726223.png)

**Comment by ronaldohoch** [comment]
This is an Omnia-editor enhancement, does it?

**Comment by TorstenDittmann** [comment]
Yes, but it is a hard design question on how to implement it and prevent cluttering the ui. This is something that I have to work on in the future, since I am not 100% of how I will implement it yet.
